rules/rule-credentials-in-env-var/test/pod-allowed-values-keys/input/pod.yaml

```
@@ -20,7 +20,17 @@ spec:
       - name : random
         value : "Hello from the environment"
       - name: AWS_TOKEN_FILE
-        value: /etc/secret-volume/aws
+        value: hunter2
+      - name: DB_PASSWORD
+        valueFrom:
+        secretKeyRef:
+        name: database-secret
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)